identity-apps-core/apps/recovery-portal/src/main/webapp/password-recovery-username-request.jsp

```diff
@@ -79,6 +79,7 @@
     layoutData.put("containerSize", "medium");
 %>
 
+<!doctype html>
```

Comment:
Let's add the lang attribute as well.

Reply:
it's added in line 83 inside the html tag. Is it necessay to add it seperately?

Reply:
No we don't. Sorry my bad I didn't see that.